acbuild/abort.go

```diff
@@ -42,11 +42,23 @@ func runAbort(cmd *cobra.Command, args []string) (exit int) {
 		return 1
 	}
 
+	err := getLock()
```

Comment:
could you actually just have this return the lock, and then pass that into the defer func. I'm not a big fan of the global variable. Sorry I didn't raise this before..